src/server/gc_worker/gc_worker.rs

```diff
@@ -1084,11 +1182,13 @@ where
             applied_lock_collector: None,
             gc_manager_handle: Arc::new(Mutex::new(None)),
             feature_gate,
+            region_info_provider,
         }
     }
 
     pub fn start_auto_gc<S: GcSafePointProvider, R: RegionInfoProvider + Clone + 'static>(
         &self,
+        kv_engine: &E::Local,
```

Comment:
@SpadeA-Tang Could you help explain a bit why this is necessary?

Reply:
Okay... I have remembered why this change exists. I tried to remove the kv_engine() from Engine trait in some point of time but I gave up this way later without changeing it back. I will propose another PR for reverting it.